block_news.php

```diff
@@ -29,19 +29,17 @@ class block_news extends block_base {
      * @return void
      */
     public function init() {
-        global $USER;
         $this->title = get_string('pluginname', 'block_news');
     }
 
     /**
      * Gets the block settings.
      *
+     * @return void
      */
     public function specialization() {
-        if (isset($this->config->title)) {
-            $this->title = $this->title = format_string($this->config->title, true, ['context' => $this->context]);
-        } else {
-            $this->title = get_string('pluginname', 'block_news');
```

Comment:
`isset($this->config->title)` is set when the value is an empty string. e.g. when user add the title and then later removed it.
I think it's better to check for empty string using `empty()`.

Removed the extra `$this->title`.

`else {
            $this->title = get_string('pluginname', 'block_alerts');
    }`
this is not necessary as it is set in the `init()` already.